phost/src/main.rs

```diff
@@ -83,7 +95,28 @@ async fn get_block_at(client: &subxt::Client<Runtime>, h: Option<u32>)
                              .ok_or(Error::BlockNotFound())?;
 
     let block = deopaque_signedblock(opaque_block);
-    Ok(block)
+
+	let block_with_events;
+	if h.is_some() && h.unwrap() > 0 {
+		block_with_events = fetch_events(&client, &block, h.unwrap()).await.unwrap();
+	} else {
+		block_with_events = BlockWithEvents {
+			block,
+			events: None,
+			proof: None,
+			key: None,
+		}
+	}
+
+	Ok(block_with_events)
+}
+
+async fn get_storage(client: &subxt::Client<Runtime>, hash: Option<Hash>, storage_key: StorageKey) -> Option<Vec<u8>> {
+	client.storage(storage_key, hash).await.expect("error when getting storage")
```

Comment:
```suggestion
	client.storage(storage_key, hash).await.expect("Error when getting storage")
```

Error messages should have a correct case